pennylane/circuit_drawer/representation_resolver.py

```diff
@@ -115,6 +115,9 @@ def single_parameter_representation(self, par):
         if isinstance(par, qml.variable.Variable):
             return par.render(self.show_variable_names)
 
+        if isinstance(par, str):
+            return par
+
```

Comment:
Be sure to add `str` in the docstring!

Just for my own understanding: so this was needed do to passing the one letter string representation of each Pauli, correct? :)

If so, maybe some reference maybe an example of `PauliRot` could also be added somehow (most of the times I'd expect `par` to be of the other, already existing types).